bot/cogs/snakes.py

```diff
@@ -28,9 +44,23 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         :return: A dict containing information on a snake
         """
+        name = str(name)
```

Comment:
No, don't do this. If there's no snake name passed, this will now be `"None"`, which is annoying to check for.